src/Loggers/MSVC/ForwardingLogger.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.Build.Framework;
+
+namespace ReferenceTrimmer.Loggers.MSVC;
+
+/// <summary>
+/// Forwarding logger that runs in each MSBuild worker node outside of the primary node.
+/// </summary>
+public sealed class ForwardingLogger : IForwardingLogger
+{
+    private readonly List<object> _loggerList = new();
```

Comment:
I'm not clear what this is for

Reply:
The class itself or the _loggerList?

Reply:
Removed _loggerList in favor of a single ref to an UnusedLibsLogger.

Reply:
Still not sure I totally get why `UnusedLibsLogger` is separate. Perhaps just to separate the MSBuild stuff from the "real" logic?

Reply:
Doesn't make s huge different (one extra allocation at worst) so I guess it's whatever.